pages/draft-06/json-schema-release-notes.md

```diff
@@ -10,41 +7,41 @@ Release notes for migrating from zyp-04 and fge-00 (draft-04) to wright-01 (draf
-`"examples"` | added | array of examples with no validation effect; the value of `"default"` is usable as an example without repeating it under this keyword
-
-#### Formats: `"uri"` vs `"uri-reference"`
+- [Q: What are the changes between draft-04 and draft-06?](#q:-what-are-the-changes-between-draft-04-and-draft-06)
+  - [Backwards-incompatible changes](#backwards-incompatible-changes)
+  - [Additions and backwards-compatible changes](#additions-and-backwards-compatible-changes)
+  - [Formats: "uri" vs "uri-reference"](#formats:-vs)
+- [Q: What happened to draft-05?](#q:-what-happened-to-draft-05)
+- [Q: What happened to all the discussions around re-using schemas with "additionalProperties"?](#q:-what-happened-to-all-the-discussions-around-re-using-schemas-with-additionalproperties)
+
+#### Q: What are the changes between draft-04 and draft-06?
+
+##### Backwards-incompatible changes
+
+| keyword                                       | change                                                                                         | consequence                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| --------------------------------------------- | ---------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `"id"`                                        | replaced by `"$id"`                                                                            | no longer easily confused with instance properties named `"id"`                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
```

Comment:
todo: in the previous tables the changes are italicized, whatever the format, please ensure it's kept consistent across documents.